test/module/EntraBeta/Get-EntraBetaObjectSetting.Tests.ps1

```diff
@@ -0,0 +1,75 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+BeforeAll {  
+    if ((Get-Module -Name Microsoft.Graph.Entra.Beta) -eq $null) {
+        Import-Module Microsoft.Graph.Entra.Beta        
+    }
+    Import-Module (Join-Path $psscriptroot "..\Common-Functions.ps1") -Force
+
+    $scriptblock = {
+        # Write-Host "Mocking Get-EntraBetaObjectSetting with parameters: $($args | ConvertTo-Json -Depth 3)"
+        return @(
+            [PSCustomObject]@{
+                id    = "bbbbbbbb-1111-2222-3333-cccccccccccc"
+                displayName       = 'Group.Unified.Guest'
+                values     = @{value=$false; name="AllowToAddGuests"}
+                templateId   = "bbbbbbbb-1111-2222-3333-cccccccccaaa"
+            }
+        )
+    }
+
+    Mock -CommandName Invoke-GraphRequest -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra.Beta
+}
+
+Describe "Get-EntraBetaObjectSetting" {
+    Context "Test for Get-EntraBetaObjectSetting" {
```

Comment:
Add test for `Get-EntraBetaObjectSetting -TargetType "Groups" -TargetObjectId "9b424169-7f2b-4747-aa64-afbecbb28111" -Property DisplayName`

Reply:
added